content/new-routing.md

```diff
@@ -0,0 +1,446 @@
+
+```js
+// the appNotFound template is used for unknown routes and missing lists
+FlowRouter.notFound = {
+  action() {
+    BlazeLayout.render('appBody', {main: 'appNotFound'});
+  }
+};
+```
+
+The second is when the URL is valid, but doesn't actually match any data. Here we
+
+XXX: we don't actually do this yet for Todos, fix https://github.com/meteor/todos/issues/53
+
+
+### Analytics
+
+It's common to want to know which pages of your app are most commonly visited, and where users are coming from. Read more about analytics in general in the _Analytics/Monitoring guide_, but here's a simple setup that will get you URL tracking using Google Analytics. We'll be using the `okgrow:analytics` package.
+
+```
+meteor add okgrow:analytics
+```
+
+- [okgrow/analytics on GitHub](https://github.com/okgrow/analytics)
+
+Now, we need to configure the package with our Google Analytics key (the package also supports a large variety of other providers, check out the documentation [on Atmosphere](https://atmospherejs.com/okgrow/analytics)). Pass it in as part of _Meteor settings_:
+
+```js
+{
+  "public": {
+    "analyticsSettings": {
+      // Add your analytics tracking id's here
+      "Google Analytics" : {"trackingId": "Your tracking ID"}
+    }
+  }
+}
+```
+
+That's it! The analytics package hooks into Flow Router and records all of the page events for you.
+
+### Server Side Routing
+
+As we've discussed, Meteor is a framework for client rendered applications, but this doesn't always remove the requirement for server rendered routes. There are two main use cases for server-side routing.
+
+#### Server Routing for API access
+
+Although Meteor allows you to [write low-level connect handlers](http://docs.meteor.com/#/full/webapp) to create any kind of API you like on the server-side, if you all you want to do is create a RESTful version of your Methods and Publications, you can often use the [`simple:rest`](http://atmospherejs.com/simple/rest) package to do this easily. See the {% link_to data-loading 'Data Loading' %} and {% link_to methods 'Methods' %} articles for more information.
+
+#### Server Rendering
+
+The Blaze UI library does not have support for server-side rendering, so it's not possible to render your pages on the server if you are using it. However, the React UI library does. This means it is possible to render HTML on the server if you use React as your rendering framework.
+
+Although Flow Router can be used to render React components more or less exactly as we've described above for Blaze, at of this writing, Flow Router's support for SSR is [still experimental](https://kadira.io/blog/meteor/meteor-ssr-support-using-flow-router-and-react). 
+
+If you want to use SSR and React with Meteor, the best approach probably revolves around using the [React Router package] XXX: fill this out
```

Comment:
Hmmm. I think to do it properly you probably end up with something like https://github.com/mystor/meteor-routecore. Hmm another Routing library. Great ;)